SmartDeviceLinkTests/ProxySpecs/SDLSecondaryTransportManagerSpec.m

```diff
@@ -52,6 +52,7 @@ @interface SDLSecondaryTransportManager ()
 @property (strong, nonatomic) NSMutableDictionary<SDLServiceTypeBox *, SDLTransportClassBox *> *streamingServiceTransportMap;
 @property (strong, nonatomic, nullable) NSString *ipAddress;
 @property (assign, nonatomic) int tcpPort;
+@property (assign, nonatomic) BOOL isAppReady;
```

Comment:
You really should add new tests for each of these issues:

1. When the security manager is set before the RAIR
2. When the security manager is set after the RAIR

Reply:
added